### ISSUE TITLE
Give indication of NVDA errors and other audiocons in speechviewer

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#6647]
This would be useful speech viewer users. Being able to detect an error when there is no sound allows more flexibility in the way NVDA is used and displaying the mode that NVDA is in could also be quite helpful.

**Comment by bhavyashah** [comment]
Whenever this gets implemented, I suggest having a visually distinctive format for the textual mentions for sounds. Otherwise, if the browse mode sound were to be represented in the Speech Viewer as 'browse mode', NVDA might well have the speech feedback for the same enabled and the sighted individual would remain unaware of the distinction, or else NVDA might just have encountered a piece of text 'browse mode' and read it (though those chances are slim). From the sounds of it, I am of the opinion that there should be something visual to distinguish those Speech Viewer contents as sounds and not mere text to speech. Just a thought that popped into my mind and I reckoned was perhaps worth noting.
